src/lint/linter/ArcanistGoVetLinter.php

```diff
@@ -62,21 +62,29 @@ protected function getMandatoryFlags() {
     return ['tool', 'vet'];
   }
 
+  protected function getDefaultMessageSeverity($code) {
+    return ArcanistLintSeverity::SEVERITY_WARNING;
+  }
+
   protected function parseLinterOutput($path, $err, $stdout, $stderr) {
     $lines = phutil_split_lines($stderr, false);
 
     $messages = array();
     foreach ($lines as $line) {
-      $matches = explode(':', $line, 3);
```

Comment:
this didn't work for me with this output 
```vet: test.go: test.go:1:9: expected 'IDENT', found 'go' (and 1 more errors)```
but I'm not sure if that's the case for other output